src/Logary/DataModel.fs

```diff
@@ -1158,19 +1158,33 @@ module Message =
   [<CompiledName "EventFatalFormat">]
   let eventFatalf fmt = Printf.kprintf (event LogLevel.Fatal) fmt
 
+  open Logary.Utils.FsMessageTemplates
+
+  /// A destructuring strategy for FsMessageTemplates which simply treats 
+  /// everything as a 'Scalar' object which can later be handled by Logary
+  /// as `Field (Value.ofObject o, None)`
+  let internal destructureEverythingAsScalarObject : Destructurer =
+    fun request -> TemplatePropertyValue.ScalarValue request.Value
+  
+  let internal captureNamesAndValuesAsScalars (t: Template) (args: obj[]) = 
+    let selfLogByIgnoring : SelfLogger = fun (t, v) -> ()
```

Comment:
@haf I was wondering about what to do with `SelfLogger` - as you can see on this line, it is currently ignored.

Take the following scenario as an example:
```fsharp
    Message.eventFormat (level, "Hello, {one} {two}", [| 1; 2; 3 |]) // oops - 2 fields <> 3 params
    |> Logger.log myLogger
    |> start
```

I've specified that there will be two parameters (`{one}` and `{two}`), however I actually provided three (`[| 1; 2; 3 |]`). Somebody should be warned about this mistake, but I'm not sure to go about that here. It looks like Logary has an `internal target` but I don't seem to have access to it here. Any ideas?

![logarynamedtemplatefields-warning](https://cloud.githubusercontent.com/assets/570470/15144572/a978e36c-16f3-11e6-827c-95f2ef0cd190.png)